services/policy-gateway/src/services/authority_decision_service.py

```
@@ -1290,13 +1331,167 @@ def _evaluate_pr_merge(
+        for p in (
+            r"^rm\s+(-[a-zA-Z]*f[a-zA-Z]*\s+|/)",  # rm -rf, rm /
+            r"^sudo\s+rm\s",  # sudo rm
+            r"^mkfs\.",  # mkfs filesystem creation
+            r"^dd\s+.*of=/dev/",  # dd to device
+            r"^:\(\)\s*\{\s*:\|:\&\s*\}\s*;",  # fork bomb
+            r"^chmod\s+(-[R]*\s+)?0?777\s",  # chmod 777
+            r"^chown\s+.*\s+/",  # chown on paths
+            r"^curl\s+.*\|\s*sh(ell)?\b",  # curl | sh
+            r"^wget\s+.*\|\s*sh(ell)?\b",  # wget | sh
+        )
+    ]
```

Ruff (0.15.10)

[warning] Mutable default value for class attribute

(RUF012)